dev/tools/gen_defaults/lib/chip_template.dart

```diff
@@ -41,21 +45,23 @@ class _${blockName}DefaultsM3 extends ChipThemeData {
   Color? get surfaceTintColor => ${colorOrTransparent("$tokenGroup.container.surface-tint-layer.color")};
 
   @override
-  Color? get checkmarkColor => ${color("$tokenGroup.with-icon.selected.icon.color")};
+  Color? get checkmarkColor => null;
```

Comment:
If we change tokenGroup to filterChip, should we here use `with-leading-icon.selected.leading-icon.color`?